appveyor.yml

```
@@ -1,36 +1,36 @@
-environment:
+# Build only the master branch, tagged commits, and pull requests
+branches:
+  only:
+    - master
+    - /\d+\.\d+.*/
+# Don't run the (redundant) branch build with a pull request
+skip_branch_with_pr: true
 
+environment:
+  PYTHONUNBUFFERED: 1
+  PYTHONIOENCODING: utf-8
+  PYTHONLEGACYWINDOWSSTDIO: 1
+  MINICONDA: C:\\Miniconda3-x64
   matrix:
-    - TARGET_ARCH: x64
-      CONDA_PY: 37
-      CONDA_INSTALL_LOCN: C:\\Miniconda-x64
-    - TARGET_ARCH: x64
-      CONDA_PY: 38
-      CONDA_INSTALL_LOCN: C:\\Miniconda3-x64
+    - PYTHON_VERSION: 3.7
+    - PYTHON_VERSION: 2.7
```

Why do you add 2.7 again? Is there a need for it? I thought Anaconda would neither support it anymore?

```suggestion
    - PYTHON_VERSION: 3.8
```

---

It's my personal preference to maintain it as long as it is not an undue burden. If you, me, or @jjhelmus have a PR that breaks on p27 and there is no easy fix, we can revisit.

---

And if we _do_ decide to drop p27, we need to reflect that in the conda recipe as well as a run constraint

---

Actually @xhochy I'll go ahead and allow python 2.7 failures in CI. There may indeed be a problem with the `tar` file format in that combination.